front_end/src/components/layout/Navbar.js

```diff
@@ -42,88 +32,122 @@ class Navbar extends Component {
 
```

Comment:
Please remove " let { classes } = this.props; "  . The line is unclickable somehow

Reply:
done